origin_ca.go

```diff
@@ -0,0 +1,146 @@
+package cloudflare
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// OriginCA resource
+// This is the origin_ca resource definition as defined here:
+// https://api.cloudflare.com/#cloudflare-ca
+type OriginCA struct {
```

Comment:
Might want to call this `Certificate` since it describes a certificate and not the CA itself. :-)